Sources/Tentacle/RepositoryInfo.swift

```diff
@@ -63,6 +60,65 @@ public struct RepositoryInfo: CustomStringConvertible, Identifiable {
     public var description: String {
         return nameWithOwner
     }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        self.id = try container.decode(ID.self, forKey: .id)
+        self.owner = try container.decode(UserInfo.self, forKey: .owner)
+        self.name = try container.decode(String.self, forKey: .name)
+        self.nameWithOwner = try container.decode(String.self, forKey: .nameWithOwner)
+        self.body = try? container.decode(String.self, forKey: .body)
```

Comment:
I think this should be `try container.decodeIfPresent`?

Reply:
Yeah, it should 😞

Reply:
Fixed.